src/effect.cpp

```diff
@@ -1294,8 +1294,19 @@ void load_effect_type( JsonObject &jo )
 
     new_etype.impairs_movement = hardcoded_movement_impairing.count( new_etype.id ) > 0;
 
+    if( jo.has_member( "flags" ) ) {
```

Comment:
```suggestion
    flags = jo.get_tags( "flags" );
```

Reply:
As far as I can see, JsonObject does not have a `get_tags` method?

Reply:
https://github.com/CleverRaven/Cataclysm-DDA/blob/master/src/json.h#L708